rerun_py/src/dataframe.rs

```diff
@@ -1382,6 +1383,36 @@ impl PyRecording {
             .as_str()
             .to_owned())
     }
+
+    fn to_stream(&self, py: Python<'_>) -> PyResult<crate::python_bridge::PyRecordingStream> {
+        let mut batcher_config = re_chunk::ChunkBatcherConfig::from_env().unwrap_or_default();
+        let on_release = |chunk| {
+            GARBAGE_QUEUE.0.send(chunk).ok();
+        };
+        batcher_config.hooks.on_release = Some(on_release.into());
+
+        let application_id = self.application_id()?;
+        let recording_id = self.store.read().id();
+
+        let recording = RecordingStreamBuilder::new(application_id)
+            .batcher_config(batcher_config)
+            .is_official_example(false)
```

Comment:
leaving that to default does the usual detection of when a thing is an example, that should be fine
```suggestion
```